pkg/api/datasources.go

```diff
@@ -286,7 +286,7 @@ func GetDataSourceIdByName(c *models.ReqContext) Response {
 // /api/datasources/:id/resources/*
 func (hs *HTTPServer) CallDatasourceResource(c *models.ReqContext) {
 	datasourceID := c.ParamsInt64(":id")
-	ds, err := hs.DatasourceCache.GetDatasource(datasourceID, c.SignedInUser, c.SkipCache)
+	ds, err := hs.DatasourceCache.GetDatasource(datasourceID, c.SignedInUser, c.SkipCache, nil)
```

Comment:
Passing `nil` like that seems less desirable. Can we think of a better way?

Reply:
@bergquist I can try to come up with something.